lighthouse-core/report/html/report-styles.css

```diff
@@ -99,19 +101,23 @@
   --color-red: #FF4E42;
   --color-white: #FFFFFF;
   --header-padding: 20px 0 20px 0;
+  --plugin-badge-bg: var(--color-white);
+  --plugin-badge-size: 36px;
```

Comment:
let's define plugin-badge-size based on circle-size via calc()

I have a plan to dynamically set circle-size to slowly shrink the badges as they scroll, so having these all using the same base numbers would help a lot.

Reply:
done